models/dataset.py

```diff
@@ -20,6 +19,7 @@ class Dataset(abc.ABC):
     icon: Optional[str]
     recommended_model_parameters: Optional[Dict[str, Dict[str, Any]]]
```

Comment:
@soheilazangeneh Can you add recommended parameters for automl to the dataset_service?

Reply:
Done. The ones I added is actually the same input I used during testing. Also for context window and horizon I am not sure if they should be the optimal values for AutoML training :-? I didn't try different values to see what is best!